collectd/files/java.conf

```diff
@@ -1,3 +1,18 @@
+{%- from "collectd/map.jinja" import collectd_settings with context %}
+{%- set java_settings = collectd_settings.plugins.java %}
+
+{%- set host = java_settings.host %}
+{%- set port = java_settings.port %}
+{%- set user = java_settings.user %}
+{%- set pass = java_settings.pass -%}
+
```

Comment:
I would go ahead and put these all inline instead of using "set". Just to make it match the rest of the files. That's my opinion, your's may be different.